health/arg_codes.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Tailscale Inc & AUTHORS
```

Comment:
filename "arg_codes.go" is kinda outdated now. Mabye just "args.go" ?

Reply:
Renamed